components/leadzen_ai/actions/simple-search/simple-search.mjs

```diff
@@ -0,0 +1,55 @@
+import app from "../../leadzen_ai.app.mjs";
+
+export default {
+  key: "leadzen_ai-simple-search",
+  name: "Simple Search",
+  description: "Performs a simple search based on the provided name.",
```

Comment:
This doesn't seem to describe what the action does. The user is meant to provide a URL instead of a name. Also, the doc link is missing.
```suggestion
  description: "Fetches detailed LinkedIn profile information based on the provided URL. [See the documentation](https://api.leadzen.ai/docs#/People/people_search_detailed_api_people_linkedin_url_profile_post)",
```